src/theme.js

```diff
@@ -35,26 +44,72 @@ module.exports = {
     3: '3px',
     4: '4px'
   },
+  gap: {
+    ...spacing
+  },
+  gridColumn: {
+    'span-auto': 'auto',
+    'span-full': '1 / -1',
+    ...Object.fromEntries(
+      gridColumnSpans.map(n => [
+        `span-${n}`,
+        `span ${n} / span ${n}`
+      ])
+    )
+  },
+  gridRow: {
+    'span-auto': 'auto',
+    'span-full': '1 / -1',
+    ...Object.fromEntries(
+      gridRowSpans.map(n => [
+        `span-${n}`,
+        `span ${n} / span ${n}`
+      ])
+    )
+  },
+  gridTemplateColumns: {
+    ...Object.fromEntries(
+      gridColumns.map(n => [
+        n,
+        `repeat(${n}, minmax(0, 1fr))`
+      ])
+    )
+  },
+  gridTemplateRows: {
+    ...Object.fromEntries(
+      gridRows.map(n => [
+        n,
+        `repeat(${n}, minmax(0, 1fr))`
```

Comment:
I'm not sure if we should make a more custom class for site purposes, but I had to modify this to e.g. `2: repeat(2, 0fr)` so that a multi-row layout wouldn't be filled with a buncha white space. might be something we wanna discuss, lmk!

Reply:
actually, might make more sense to do `n: repeat(n, minmax(0, auto))`?